tensorflow/lite/examples/label_image/get_top_n_impl.h

```diff
@@ -30,19 +32,29 @@ extern bool input_floating;
 template <class T>
 void get_top_n(T* prediction, int prediction_size, size_t num_results,
                float threshold, std::vector<std::pair<float, int>>* top_results,
-               bool input_floating) {
+               TfLiteType input_type) {
```

Comment:
This is causing issues in `label_image_test.cc`, where some tests pass in a bool at parameter no. 6. So you will probably have to modify that file.

Reply:
Thanks. Fixed.